controls/SV-204488.rb

```diff
@@ -22,11 +26,11 @@
   tag severity: 'medium'
   tag gtitle: 'SRG-OS-000480-GPOS-00227'
   tag gid: 'V-204488'
-  tag rid: 'SV-204488r603261_rule'
+  tag rid: 'SV-204488r861006_rule'
   tag stig_id: 'RHEL-07-021040'
   tag fix_id: 'F-4612r88657_fix'
   tag cci: ['CCI-000318', 'CCI-000368', 'CCI-001812', 'CCI-001813', 'CCI-001814']
-  tag nist: ['CM-3 f', 'CM-6 c', 'CM-11 (2)', 'CM-5 (1)', 'CM-5 (1)']
+  tag nist: ['CM-3 f', 'CM-6 c', 'CM-11 (2)', 'CM-5 (1)', 'CM-5 (1) (a)']
```

Comment:
Comment for unchanged code below:

Rename variable `inv_mode` to describe the purpose of this variable. What is the purpose?

Reply:
Renamed the variable name after talking to Aaron